### PR TITLE
Update eip2537: Add link to noble library.

### PR DESCRIPTION
**Pull request by paulmillr**
It's very useful to have it, because noble is the only library that has all possible optimizations of final exponentiation used for pairings. Go and Rust codes don't have all of 'em. Also, it's the only JS implementation. 

https://github.com/paulmillr/noble-bls12-381

### CODE REVIEW

**Comment by MicahZoltu** [comment]
This EIP had a typo in it that somehow got merged previously, which blocked CI.  I have updated this PR to include a fix to the typo in hopes that it gets CI to move forward.

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 2537 requires approval from one of (@shamatar)

**Comment by axic** [comment]
@shamatar can you please review this?

**Comment by shamatar** [comment]
Thank you for pinging me, Alex.

Can someone with write access merge it?

**Comment by axic** [comment]
If you github approve, the bot will merge it. If it doesn't work I'll do it, but please try.

**Review by shamatar — APPROVED** [review]

**Comment by shamatar** [comment]
Done, let's check if bot will do everything.

**Pull request merged**
